assets/app/view/game/round/stock.rb

```diff
@@ -350,6 +354,16 @@ def render_buy_input_interval(company)
             *buy_buttons,
           ])]
         end
+
+        def render_bid_input(company)
+          return [] if @step.max_bid(@current_entity, company) < @step.min_bid(company)
+          return [] if @current_actions.include?('bidding_tokens') &&
+            @step.highest_player_bid?(@current_entity, company)
```

Comment:
Is this true, you can't increase your bid if winning? I understand why you wouldn't want to, but is it against the rules?

Reply:
Yes it is

Reply:
Yes you cant move the cude if you are the highest:

4.9.1 Move a previously placed bidding token to a higher-valued empty space on the same item. This is only allowed if
another player has placed a higher bid.